roofit/roofitcore/src/RooFuncWrapper.cxx

```diff
@@ -136,3 +165,53 @@ void RooFuncWrapper::gradient(const double *x, double *g) const
 
    _grad(const_cast<double *>(x), _observables.data(), g);
 }
+
+void RooFuncWrapper::buildCode(RooAbsReal const &head, std::string &code, RooArgSet const & /* paramSet */,
+                               RooArgSet const &ObsSet, const RooAbsData *data)
+{
+   RooFit::Detail::CodeSquashContext ctx;
+
+   // First update the result variable of params in the compute graph to in[<position>].
+   int idx = 0;
+   for (auto *param : static_range_cast<RooRealVar *>(_params)) {
```

Comment:
```suggestion
   for (RooAbsArg *param : _params) {
```
The cast is not necessary here (but without the cast I would not use `auto` because otherwise the type is not immediately clear).